anda/fonts/maple/maple-fonts.spec

```diff
@@ -0,0 +1,44 @@
+%define debug_package %nil
+%define _ttfontsdir %{_datadir}/fonts/maple
+
+Name:           maple-fonts
+Version:        6.4
+Release:        %autorelease
+Summary:        Open source monospace & nerd font with round corner and ligatures.
+License:        OFL-1.1
+URL:            https://github.com/subframe7536/Maple-font
+Source0:        %{url}/releases/download/v%{version}/MapleMono-SC-NF.zip
+Source1:        %{url}/releases/download/v%{version}/MapleMono-ttf.zip
+Source2:        %{url}/releases/download/v%{version}/MapleMono-NF.zip
+Source3:        https://raw.githubusercontent.com/subframe7536/maple-font/main/OFL.txt
+BuildArch:      noarch
+BuildRequires:  unzip
+ 
+%description
+%summary
+
+%prep
+%setup -q -c -n %{name}-%{version}
+unzip %{S:1}
+unzip %{S:2}
+cp %{S:3} .
+ 
+%build
+ 
+%install
+install -d %{buildroot}%{_ttfontsdir}
+# by default install command uses 755 umask
+install -m 644 *.ttf %{buildroot}%{_ttfontsdir}
+ 
+%post
```

Comment:
You shouldn't have to specify %post and %postun, since you aren't doing anything here anyways.